modules/exploits/linux/http/cisco_rv32x_rce.rb

```diff
@@ -0,0 +1,164 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::Remote::HttpServer::HTML
+  include Msf::Exploit::CmdStager
+
+  def initialize(info={})
+    super(update_info(info,
+      'Name'           => "Cisco RV320 and RV325 Unauthenticated Remote Code Execution",
+      'Description'    => %q{
+This exploit module combines an information disclosure (CVE-2019-1653)
```

Comment:
The module description text is traditionally indented one additional level to the `Description` hash key.